### DISCUSSION TITLE
Analytics or log parser

### DISCUSSION THREAD

**Discussion by jessec** [docker-mailserver/docker-mailserver#3678] [General]
Does anybody know a solution to get analytics out of the server? I saw a few solutions but they seem very old and not maintained.

**Comment by casperklein** [comment]
Take a look at pflogsumm/logwatch: https://docker-mailserver.github.io/docker-mailserver/latest/config/environment/#reports

**Reply by jessec** [reply]
OK I somehow missed that. Thanks
